### ISSUE TITLE
simplify cranking ignition options

### ISSUE THREAD

**Issue by mck1117** [rusefi/rusefi#2713]
- maybe remove useAdvanceCorrectionsForCranking
- maybe remove useSeparateAdvanceForCranking

**Comment by rusefillc** [comment]
@andreika-git FYI

**Comment by andreika-git** [comment]
/firmware/config/boards/proteus/prepend.txt
+#define ts_show_advanced_cranking_settings false

**Comment by mck1117** [comment]
Yes of course we could just hide it, but that isn't the point. There's no good reason I can think of to enable ignition corrections (Which could easily be +-20 degrees) while cranking.

**Comment by andreika-git** [comment]
When starting a very cold engine, the fuel mixture is extremely rich, besides the gasoline evaporates slowly at low temperatures, and thus more liquid gasoline comes into the combustion chamber. Liquid fuel burns really slowly, so more early ignition is needed to burn it completely. Another reason is that when using separate advance tables for cranking/idling, there can be a significant difference when transitioning from cranking curve to idle (due to corrections disabled for cranking), which leads to noticeable engine wobbling at cranking rpm threshold.

As for the "huge 20 degree" corrections, here is the example of CLT correction curve tested on at least 2 different real cars (in winter) and currently used:
![image](https://user-images.githubusercontent.com/15104768/118488605-93203800-b724-11eb-96d8-de223db64c98.png)
Even at -40C it adds only 8 degrees. If someone has a different curve with more aggressive angles, then... well, just don't enable the correction for cranking.
The second one, the IAT correction, usually doesn't affect the cold start at all:
![image](https://user-images.githubusercontent.com/15104768/118488887-dda1b480-b724-11eb-9ba1-714e5ee125ec.png)
The third fsioTimingAdjustment correction is very user-specific and cannot be taken into consideration.
Finally, the last pidTimingCorrection is disabled for cranking anyway (if (phase != Phase::Idling) return 0).

**Comment by mck1117** [comment]
> so more early ignition is needed to burn it completely

OEM tunes do not necessarily reflect this:

![image](https://user-images.githubusercontent.com/568254/118529882-db296580-b6f8-11eb-823d-0a66e196801e.png)

Top two tables control open loop running ECT correction.  Bottom table is cranking.  Essentially no change to cranking advance, except when very hot.

> The second one, the IAT correction, usually doesn't affect the cold start at all:

That's not the default table.  Default table has +4 degrees in the bottom left corner, and 3 degrees at 10 C.  You could very easily end up with an accidental extra 8-12 degrees of timing, which on my engine is enough to get starter kickback.

**Comment by blundar** [comment]
Cranking is its own beast.  Closed loop spark corrections appropriate for idle or general operation would not be good in the least.  There is a very good reason that crank operation hands off to run operation at a pretty low RPM, quite a bit lower in many OE situations that we do.
Cranking spark vs, ECT = good.
Using the same timing scale as normal operation... meh

**Comment by blundar** [comment]
Granted much is IMHO, but its a method I've honed over decades.
https://rusefi.com/forum/viewtopic.php?f=16&t=2021

**Comment by mck1117** [comment]
>  Closed loop spark corrections

he's right here, this one does actually disable itself while cranking
